src/biblenotify/ui/BibleNotify/Views/HomeView.qml

```diff
@@ -37,9 +37,12 @@ Item {
                 Layout.alignment: Qt.AlignHCenter
                 Layout.maximumWidth: root.width - root.anchors.leftMargin - root.anchors.rightMargin
                 horizontalAlignment: Text.AlignHCenter
-                font { pixelSize: 20; italic: true; }
+                font { 
+                    pixelSize: 20; 
+                    italic: true; 
```

Comment:
Semi-colons not needed here.

Reply:
True (again, leftover from the copy-paste from the docs)

Reply:
Just committed removing these.